docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<h2>Write the function template</h2><p> </p><p>We will write a function template for Hamiltonian simulation using the AQC-Tensor Qiskit addon to map the problem description to a reduced-depth circuit for execution on hardware.</p><p> </p><p>We will save the following code to&nbsp;<code>./source_files/hamsim_template.py</code>. This file is the function template we'll upload to and run remotely with Qiskit Serverless.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>